docs/configuration/images.md

```diff
@@ -16,4 +16,70 @@ FIXME: add content
 
 ## LLNL Import Tool
 
-FIXME: add content
+The LLNL workflow has its own import tool designed to allow one or more images to be loaded, transformed and cropped before loading them into the GUI for analysis. To get started, launch the tool by going to `File->Import->LLNL Import Tool`.
+
+![LLNL Import Menu Option](img/menu_llnl_import.png)
+
+Once the import tool has opened you have the option to dock it in the GUI or allow it to remain as a standalone dialog.
+
+![Dock LLNL Import Tool](img/dock_llnl_import.gif)
+
+### Instrument Configuration
+
+To start the workflow, select the instrument that you will be loading image data for. Once you've selected an instrument you will need to select what config you will be using for the raw data. This config file will need to include default information about all of the expected detectors. See an example of the defaults we provide for [TARDIS](https://github.com/HEXRD/hexrd/blob/master/hexrd/resources/tardis_reference_config.yml) or [PXRDIP](https://github.com/HEXRD/hexrd/blob/master/hexrd/resources/pxrdip_reference_config.yml).
+
+*Note*: Any unused detectors will simply be ignored on completion.
+
+![LLNL Instrument and Config Selection](img/instrument_and_config.png)
+
+You'll notice that there are a few options for how to load a config:
+
+1. **Current Configuration**: This will used the currently loaded instrument config file.
+2. **Load Configuration**: This will enable the ability to either use the default configuration file that we provide or to load your own. Uncheck the `Default` box to enable file loading.
+
+If you are unsure what to use at first start with the default config. For more information on configuration files see the [instrument](instrument.md) section of the docs.
+
+### Raw Image
+
+Once the instrument and configuration have been selected the `Raw Image` portion of the form will be enabled and the list of available detectors will be populated. The detector selected from the drop-down menu will determine which template will be added to the image, as well as what default transform is suggested when the image is loaded. Use the `Select Image` button to load an image.
+
+![LLNL Image Load With Deafult Transform](img/select_image_llnl.png)
+
+If there is a default transform associated with the current instrument or specific detector this will automatically be set in the load image dialog. This can still be changed either on load or in the import tool after the image is loaded.
```

Comment:
```suggestion
If there is a default transform associated with the current instrument or selected detector, this will automatically be set in the load image dialog. This can still be changed either on load or in the import tool after the image is loaded.
```